src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Functions.Cli.Interfaces;
+using Colors.Net;
+using DurableTask.AzureStorage;
+using DurableTask.Core;
+using DurableTask.Core.History;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using static Colors.Net.StringStaticMethods;
+
+namespace Azure.Functions.Cli.Common
+{
+    internal class DurableManager : IDurableManager
+    {
+        private readonly AzureStorageOrchestrationService _orchestrationService;
+
+        private readonly TaskHubClient _client;
+
+        private const string TaskHubName = "DurableFunctionsHub";
+
+        public DurableManager(ISecretsManager secretsManager)
+        {
+            var connectionString = secretsManager.GetSecrets().FirstOrDefault(s => s.Key.Equals("AzureWebJobsStorage", StringComparison.OrdinalIgnoreCase)).Value;
```

Comment:
Some customers will want to use non-default connection strings since we advertise that as a best practice. Can we update our commands to support this? My thinking is that we should look at the `host.json` file for the correct connection string name. See [here](https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-bindings#host-json) for documentation. (WARNING: Make sure we're using the v2 syntax where `"durableTask"` lives inside the `"extensions"` property. This is not shown in the preceding link at the time of writing).